integration-tests/mockApis/whereabouts.js

```diff
@@ -294,4 +294,24 @@ module.exports = {
         jsonBody: response,
       },
     }),
+  stubLocationGroups: locationGroups =>
```

Comment:
There's a stub called `stubGroups` which I think might be doing something very similar? Not that I mind :D

Reply:
Ah you might be right! I'll fix it on the next PR